src/control/cmd/daos/acl.go

```diff
@@ -316,12 +391,25 @@ func (cmd *containerSetOwnerCmd) Execute(args []string) error {
 		defer C.free(unsafe.Pointer(ap.group))
 	}
 
-	rc := C.cont_set_owner_hdlr(ap)
+	rc := C.daos_cont_set_owner(ap.cont, ap.user, ap.group, nil)
```

Comment:
We should probably be gradually phasing out the use of the `cmd_args_s` struct. In [my PR](https://github.com/daos-stack/daos/pull/11847), I'm still using it for the `ap.cont` field, but otherwise I'm providing new local variables directly to the C functions.

Reply:
Oops, I did that in most cases, just forgot in the cleanup here. I'll fix it.